Header header
float64 left_vel
float64 right_vel
float64 duration
